terraform/dns/README.md

```diff
@@ -0,0 +1,77 @@
+# DNSimple Terraform DNS Demo
+
+With the DNSimple Terraform Provider, you can manage your DNS infrastructure, including zones and records, directly within Terraform. This allows you to leverage the simplicity and power of Terraform's Infrastructure-as-Code design for your DNSimple resources, and coordinate them right alongside your other Terraform resources that form your infrastructure.
+
+To provide an example of the power, simplicity, and functionality of managing DNS using the DNSimple Terraform Provider, we've provided this demo that you can review, run, and play around with. We'll implement a basic email inbox service that has a simple but relatable design and structure of a full stack application with all the typical infrastructure:
```

Comment:
```suggestion
As an example of the power, simplicity, and functionality of managing DNS using the DNSimple Terraform Provider, we've created this demo that you can review, run, and play around with. We'll implement a basic email inbox service with a simple but relatable design and structure of a full stack application with all the typical infrastructure:
```

adjusts to avoid repeating the same word multiple times